AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -143,7 +143,7 @@ ibelieve.com##.my-2
 ibelieve.com##.w-full > div.flex > div.mx-auto.min-h-\[250px\]
 ibelieve.com#?#.bg-white:has(> div.mx-auto iframe[title*="iBelieve Daily Prayers"])
 upi.com###arc_email_subsc
-calmatters.org,berkeleyside.org##.newspack-popup-container
+thecity.nyc,calmatters.org,berkeleyside.org##.newspack-popup-container
```

Comment:
Blocks along with subscription donation banners and related articles. Although these are also annoyances, but I think that for the convenience of users it will be necessary to divide them into different filters. Or move the rule to annoyances --> other